Sources/DataPipeline/CustomerIO.swift

```diff
@@ -21,21 +21,31 @@ public extension CustomerIO {
 
         let sdkConfig = SdkConfig.Factory.create(siteId: "", apiKey: "", region: .US)
         let newDiGraph = DIGraph(sdkConfig: sdkConfig)
-        initializeSharedInstance(with: implementation, diGraph: newDiGraph)
+        initialize(implementation: implementation, diGraph: newDiGraph)
     }
 
-    #if DEBUG
-    /// Initializer for Integration Tests to update the DataPipeline instances.
-    /// To be used for testing purposes only.
-    static func initializeIntegrationTestsInstance(diGraph: DIGraph, moduleConfig: DataPipelineConfigOptions) {
-        let implementation = DataPipeline.createAndSetSharedTestInstance(diGraphShared: DIGraphShared.shared, config: moduleConfig)
+    /**
+     Common initialization method for setting up the shared `CustomerIO` instance.
+     This method is intended to be used by both actual implementations and in tests, ensuring that tests closely mimic the real-world implementation.
+     */
+    private static func initialize(implementation: DataPipelineInstance, diGraph: DIGraph) {
```

Comment:
Added this to make sure both tests and actual SDK is initialized using same path.